playground/src/assembleDecorations.ts

```diff
@@ -0,0 +1,156 @@
+import * as tsdoc from '@microsoft/tsdoc';
+import { ISyntaxDecoration } from './MonacoWrapper';
+import { syntaxColors } from './syntaxColors';
+
+export function assembleDecorations(decorations: ISyntaxDecoration[], docNode: tsdoc.DocNode): void {
+  switch (docNode.kind) {
+    case 'BlockTag':
+      if (docNode instanceof tsdoc.DocNodeLeaf && docNode.excerpt) {
+          switch (docNode.excerpt.content.toString()) {
```

Comment:
If the `kind=BlockTag`, then you can typecast directly to `DocBlockTag` and read `DocBlockTag.tagNameWithUpperCase`